packages/components/src/button/src/CrossButton.tsx

```diff
@@ -1,5 +1,3 @@
-import "./CrossButton.css";
```

Comment:
[Question] Just wondering if there was places where we imported CSS in the component